Source/Cellular/Cell_Basics/Cell_Basics.csproj

```diff
@@ -6,7 +6,7 @@
     <AssemblyName>App</AssemblyName>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Meadow.F7" Version="*" />
+    <ProjectReference Include="..\..\..\..\Meadow.Core\source\implementations\f7\Meadow.F7\Meadow.F7.csproj" />
```

Comment:
I think that we should keep <PackageReference Include="Meadow.F7" Version="*" />, right @jorgedevs

Reply:
if its goint to develop, it has to be a project reference